src/datastore.hpp

```diff
@@ -111,8 +111,9 @@ class datastore {
 	static b_fs::path get_parent_path(t_datastore file_type,
 								 const std::string &filename);
 
-
+	static void set_home(const std::string &home);
```

Comment:
comment, especially the special behaviour that empty-string means "use HOME from env variables".

///< Set the home directory e.g. "/home/bob/". If given empty string "", then it will set the auto mode (e.g. take env HOME)